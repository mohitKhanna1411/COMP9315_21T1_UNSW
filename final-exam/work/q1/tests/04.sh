./pages 3 5 1,2,3,1,2 <<_Q_
a,b,c
?,?,c
?,b,?
a,?,?
_Q_
