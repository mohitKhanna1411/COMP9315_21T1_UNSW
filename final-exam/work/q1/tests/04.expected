CV = < (1,0) (2,0) (3,0) (1,1) (2,1) >
query> q[1] = 00101111 11100001 00011010 01110001
q[2] = 11101011 01100101 01010001 10111010
q[3] = 01101100 01000111 11011011 00100111
Pages: 21
query> q[1] = ??
q[2] = ??
q[3] = 01101100 01000111 11011011 00100111
Pages: 4 5 6 7 12 13 14 15 20 21 22 23 28 29 30 31
query> q[1] = ??
q[2] = 11101011 01100101 01010001 10111010
q[3] = ??
Pages: 16 17 20 21 24 25 28 29
query> q[1] = 00101111 11100001 00011010 01110001
q[2] = ??
q[3] = ??
Pages: 1 3 5 7 17 19 21 23
query> 
